src/main/java/com/hartwig/platinum/kubernetes/JobSubmitter.java

```diff
@@ -48,20 +47,15 @@ public boolean submit(final PipelineJob job) {
     }
 
     private boolean submit(final PipelineJob job, final JobSpec spec) {
-        try {
-            kubernetesClientProxy.jobs()
-                    .create(new JobBuilder().withNewMetadata()
-                            .withName(job.getName())
-                            .withNamespace(NAMESPACE)
-                            .endMetadata()
-                            .withSpec(spec)
-                            .build());
-            LOGGER.info("Submitted [{}]", job.getName());
-            return true;
-        } catch (KubernetesClientException e) {
-            kubernetesClientProxy.authorise();
-            return submit(job, spec);
-        }
+        kubernetesClientProxy.jobs()
+                .create(new JobBuilder().withNewMetadata()
+                        .withName(job.getName())
+                        .withNamespace(NAMESPACE)
+                        .endMetadata()
+                        .withSpec(spec)
+                        .build());
+        LOGGER.info("Submitted [{}]", job.getName());
+        return true;
```

Comment:
since this can never be false now maybe not that useful to return?

Reply:
Fair point. I'll clean it up.